src/common/consts/contracts.ts

```diff
@@ -52,19 +52,26 @@ export const EXTENSIBLE_FALLBACK_HANDLER_CONTRACT_ADDRESS = mapAddressToSupporte
  */
 export const COMPOSABLE_COW_CONTRACT_ADDRESS = mapAddressToSupportedNetworks(COMPOSABLE_COW)
 
-export const ETH_FLOW_ADDRESS = '0xba3cb449bd2b4adddbc894d8697f5170800eadec'
-export const BARN_ETH_FLOW_ADDRESS = '0x04501b9b1d52e67f6862d157e00d13419d2d6e95'
+const ETH_FLOW_ADDRESS = '0xba3cb449bd2b4adddbc894d8697f5170800eadec'
+const BARN_ETH_FLOW_ADDRESS = '0x04501b9b1d52e67f6862d157e00d13419d2d6e95'
+// Lens does not uses the same deterministic address for the EthFlow contract as other chains.
+const ETH_FLOW_ADDRESS_LENS = '0x5A5b8aE7a0b4C0EAf453d10DCcfbA413f07ebdC2'
+const BARN_ETH_FLOW_ADDRESS_LENS = '0xFb337f8a725A142f65fb9ff4902d41cc901de222'
 
 /**
- * An object containing the addresses of ETH flow contracts for each supported chain.
- * @deprecated use ETH_FLOW_ADDRESS instead
+ * An object containing the addresses of ETH flow contracts for each supported chain for production.
  */
-export const ETH_FLOW_ADDRESSES: Record<SupportedChainId, string> = mapAddressToSupportedNetworks(ETH_FLOW_ADDRESS)
+export const ETH_FLOW_ADDRESSES: Record<SupportedChainId, string> = {
```

Comment:
I recently added `ETH_FLOW_ADDRESS` in CoW Swap.
We will need to change it when bump the SDK version:

https://github.com/cowprotocol/cowswap/pull/6138/files#diff-15b37acb56ec1190c9be0cc240d28ad3a2c426361f07e428929c22040190528bR50

Reply:
The problem is that due to some deployment issues, Lens will have a different address :(
I had to revert to the old map method